### PR TITLE
Add 1337x search plugin

### PR DESCRIPTION
**Pull request by ngosang**
Based on @v1k45 source code https://github.com/v1k45/1337x-qBittorrent-search-plugin

Changes:
* Fix download torrent => magnet
* Fix error with ',' in size
* Code simplification
* Compatible with python2, python3 and last qbittorrent release

### CODE REVIEW

**Comment by v1k45** [comment]
Does this change only affects the newer releases or old ones too? AFAIK, old plugin engine versions don't support magnet links.

**Comment by ngosang** [comment]
@v1k45 affects all qbittorrent versions, users with old versions too. It should work in all versions.

**Comment by ngosang** [comment]
@Chocobo1 could you do the pep8 fixes for me?

**Comment by Chocobo1** [comment]
>@Chocobo1 could you do the pep8 fixes for me?

OK, but it will be later, short on free time right now...

**Comment by ngosang** [comment]
I did, all PRs are ready to review.

File: nova/engines/versions.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 btdb: 1.04
 eztv: 1.01
+leetx: 2.00
```
[the remark this answers is not in the document]
Comment (ngosang) [reply nova/engines/versions.txt]: The file name must be the same as the class name and the class name can't start with numbers.

[the remark this answers is not in the document]
Comment (Chocobo1) [reply nova/engines/versions.txt]: ok, no more question from me, proceed as you see fit.

**Pull request merged**